assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -49,12 +49,16 @@ export const GalleryFrontend = ({
         </header>
       }
       {attributes.gallery_block_description &&
-        <div className="page-section-description" dangerouslySetInnerHTML={{__html: attributes.gallery_block_description}} />
+        <p className="page-section-description" dangerouslySetInnerHTML={{__html: attributes.gallery_block_description}} />
       }
 
-      {layout === 'slider' && <GalleryCarousel onImageClick={openLightbox} images={images} />}
-      {layout === 'three-columns' && <GalleryThreeColumns onImageClick={openLightbox} images={images} postType={postType} />}
-      {layout === 'grid' && <GalleryGrid onImageClick={openLightbox} images={images} />}
+      {images.length ? (
```

Comment:
I added this to avoid render components if the length of images is empty.

Reply:
Maybe we should hide the whole block if there are no images? 🤔

Reply:
Sounds logic! But there is a text and description also included